<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>InspectorMovieWidget</class>
 <widget class="QWidget" name="InspectorMovieWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>265</width>
    <height>354</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransition">
     <item>
      <spacer name="horizontalSpacerTransition">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>29</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransition">
       <property name="text">
        <string>Transition</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTransition">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTransitionDuration">
     <item>
      <spacer name="horizontalSpacerTransitionDuration">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>35</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTransitionDuration">
       <property name="text">
        <string>Duration</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxTransitionDuration">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm1">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTween">
     <item>
      <spacer name="horizontalSpacerTween">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>48</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTween">
       <property name="text">
        <string>Tween</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxTween">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutDirection">
     <item>
      <spacer name="horizontalSpacerDirection">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>34</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelDirection">
       <property name="text">
        <string>Direction</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QComboBox" name="comboBoxDirection">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutSeekLength">
     <item>
      <spacer name="horizontalSpacerSeekLength">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>58</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelSeek">
       <property name="toolTip">
        <string>Start playback at this frame</string>
       </property>
       <property name="text">
        <string>Seek</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxSeek">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm2">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLength">
     <item>
      <spacer name="horizontalSpacerLength">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>46</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelLength">
       <property name="toolTip">
        <string>Stop playback at this frame relative to the seek frame</string>
       </property>
       <property name="text">
        <string>Length</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSpinBox" name="spinBoxLength">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximum">
        <number>999999999</number>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="labelFrm3">
       <property name="text">
        <string>frm</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutLoop">
     <item>
      <spacer name="horizontalSpacerLoop1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>55</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelLoop">
       <property name="text">
        <string>Loop</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxLoop">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerLoop2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutFreezeOnLoad">
     <item>
      <spacer name="horizontalSpacerFreezeOnLoad1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>5</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelPauseOnLoad">
       <property name="text">
        <string>Freeze on load</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxFreezeOnLoad">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerFreezeOnLoad2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutTriggerOnNext">
     <item>
      <spacer name="horizontalSpacerTriggerOnNext1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>2</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelTriggerOnNext">
       <property name="text">
        <string>Trigger on next</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxTriggerOnNext">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerTriggerOnNext2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayoutAutoPlay">
     <item>
      <spacer name="horizontalSpacerAutoPlay1">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>32</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="labelAutoPlay">
       <property name="text">
        <string>Auto play</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="checkBoxAutoPlay">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacerAutoPlay2">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item>
    <spacer name="verticalSpacerMedia">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>10</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>checkBoxLoop</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>loopChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>193</y>
    </hint>
    <hint type="destinationlabel">
     <x>220</x>
     <y>85</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTransition</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>transitionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>165</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>223</x>
     <y>15</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxDirection</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>directionChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>105</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>105</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxSeek</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>seekChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>126</x>
     <y>141</y>
    </hint>
    <hint type="destinationlabel">
     <x>22</x>
     <y>170</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboBoxTween</sender>
   <signal>currentIndexChanged(QString)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>tweenChanged(QString)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>169</x>
     <y>85</y>
    </hint>
    <hint type="destinationlabel">
     <x>1</x>
     <y>141</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxFreezeOnLoad</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>freezeOnLoadChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>110</x>
     <y>221</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>209</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxTriggerOnNext</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>triggerOnNextChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>109</x>
     <y>249</y>
    </hint>
    <hint type="destinationlabel">
     <x>225</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxLength</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>lengthChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>207</x>
     <y>169</y>
    </hint>
    <hint type="destinationlabel">
     <x>239</x>
     <y>162</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkBoxAutoPlay</sender>
   <signal>stateChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>autoPlayChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>112</x>
     <y>277</y>
    </hint>
    <hint type="destinationlabel">
     <x>240</x>
     <y>303</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinBoxTransitionDuration</sender>
   <signal>valueChanged(int)</signal>
   <receiver>InspectorMovieWidget</receiver>
   <slot>transitionDurationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>47</y>
    </hint>
    <hint type="destinationlabel">
     <x>237</x>
     <y>46</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>transitionDurationChanged(int)</slot>
  <slot>loopChanged(int)</slot>
  <slot>transitionChanged(QString)</slot>
  <slot>directionChanged(QString)</slot>
  <slot>seekChanged(int)</slot>
  <slot>lengthChanged(int)</slot>
  <slot>tweenChanged(QString)</slot>
  <slot>resetTransition(QString)</slot>
  <slot>resetDuration(QString)</slot>
  <slot>resetTween(QString)</slot>
  <slot>resetDirection(QString)</slot>
  <slot>resetSeek(QString)</slot>
  <slot>resetLength(QString)</slot>
  <slot>resetLoop(QString)</slot>
  <slot>freezeOnLoadChanged(int)</slot>
  <slot>resetFreezeOnLoad(QString)</slot>
  <slot>resetUseAuto(QString)</slot>
  <slot>useAutoChanged(int)</slot>
  <slot>resetTriggerOnNext(QString)</slot>
  <slot>triggerOnNextChanged(int)</slot>
  <slot>autoPlayChanged(int)</slot>
 </slots>
</ui>
